iota/transaction/creation.py

```diff
@@ -387,6 +505,20 @@ def sign_inputs(self, key_generator):
         # type: (KeyGenerator) -> None
         """
         Sign inputs in a finalized bundle.
+
+        Generates the necessary cryptographic signatures to authorize spending
+        the inputs. You do not need to invoke this method if the bundle does
+        not contain any transactions that spend IOTAs.
```

Comment:
```suggestion
        not contain any transactions that spend iotas.
```

Reply:
❔ Should we call out the bit about not needing to invoke the method in a `.. note::` or similar?